src/sklearn_evaluation/plot/roc.py

```diff
@@ -4,11 +4,10 @@
 from sklearn.preprocessing import label_binarize
 from ..telemetry import SKLearnEvaluationLogger
 from ..util import is_column_vector, is_row_vector
-from ..plot.plot import Plot
 from sklearn_evaluation import __version__
 import json
 from pathlib import Path
-from warnings import warn  # noqa
+from ..plot.plot import AbstractPlot, AbstractComposedPlot
```

Comment:
we're no longer using relative imports, please change

Reply:
fixed